src/handwritten/task.py

```diff
@@ -405,34 +405,31 @@ def is_task_done(self):
 
     def perform_strain_shunt_cal(
             self, channel="", shunt_resistor_value=10000,
-            shunt_resistor_location=0, shunt_resistor_select=0,
-            shunt_resistor_source=0, skip_unsupported_channels=False):
+            shunt_resistor_location=ShuntElementLocation.R1, shunt_resistor_select=ShuntCalSelect.A,
+            shunt_resistor_source=ShuntCalSource.BUILT_IN, skip_unsupported_channels=False):
```

Comment:
Please match the LabVIEW VI's default values: **100000**, **R3**, A, **DEFAULT**
![image](https://github.com/ni/nidaqmx-python/assets/17914399/0a865608-ef5d-4572-8b33-64c3b6b06174)